tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -980,6 +983,26 @@ def test_detects_consecutive_db_spans(self):
             ]
         )
 
+    def test_detects_file_io_main_thread(self):
+        file_io_event = EVENTS["file-io-on-main-thread"]
+        sdk_span_mock = Mock()
+
+        _detect_performance_problems(file_io_event, sdk_span_mock)
+
+        assert sdk_span_mock.containing_transaction.set_tag.call_count == 5
+        sdk_span_mock.containing_transaction.set_tag.assert_has_calls(
+            [
+                call("_pi_all_issue_count", 1),
+                call("_pi_sdk_name", "sentry.java.android.timber"),
+                call("_pi_transaction", "c119e45a9d724b1891df4651ebf9e6db"),
+                call(
+                    "_pi_file_io_main_thread_fp",
+                    "1-GroupType.PERFORMANCE_FILE_IO_MAIN_THREAD-153198dd61706844cf3d9a922f6f82543df8125f",
+                ),
+                call("_pi_file_io_main_thread", "054ba3a374d543eb"),
+            ]
+        )
+
```

Comment:
This style of test is a vestige of the before times, when we didn't return `PerformanceProblem` instances from the detector! Would be even nicer here to do something like this:

```python
problems = _detect_performance_problems(EVENTS["file-io-on-main-thread"], sdk_span_mock)

assert problems == [
  PerformanceProblem(
    fingerprint="yadda yadda",
    op="etc"
]
```

like `test_n_plus_one_extended_detection_no_parent_span`

Reply:
Nice, I thought it was strange that we just check the tag mock but wanted to match existing tests

Reply:
Nevermind, can't do this test format until I add the option which based on Matt's comment earlier i've removed for now

Reply:
Ack, I thought that might be the case, sorry I didn't check ahead of time!